### DISCUSSION TITLE
[Feature] Building the timeline should show progressbar instead of spin

### DISCUSSION THREAD

**Discussion by mingqlin** [immich-app/immich#7181] [Feature Request]
### The feature

Since the building timeline is now paging every 5000, I would recommend we show a progress bar instead of spinning widget. For a very large asset, it help user understand how long it will take to finish building timeline

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by williewillus** [comment]
+1, this is really unintuitive and made me think the app is just broken

**Comment by raoolio** [comment]
I was going to suggest the same thing, I have a large library of photos around 60K, and when the server is building the timeline thumbnails, it would be really useful to get a glimpse of the actual progress instead of an animation. The only way to get something close to a "progress" is logging in as the admin and going down the Jobs section.
